### PR TITLE
6L Training

### PR DESCRIPTION
**Pull request by rhemon**
4800 - 4800 - 4800 - 4800 - 4800 - 4
2000 iterations
0.008 learning rate

### CODE REVIEW

**Comment by rhemon** [comment]
Closing now. Will try 5L & 6L after testing 3L & 4L with smaller layer sizes and after applying softmax in output layer.
